databases/db.js

```diff
@@ -91,5 +117,5 @@ const random = sequelize.random();
 const { Op } = Sequelize;
 
 module.exports ={
-    User, Sticker, Event, Promotion, ad, game, team, random, Op, inventory
+    User, Sticker, Event, ad, game, team, random, Op, inventory, Warehouse, Promotion
```

Comment:
```suggestion
    User, Sticker, Event, Ad, Game, Team, Random, Op, Inventory, Warehouse, Promotion
```
Modelos en masyucula pls

Reply:
RANDOM NO ES UN MODELO

Reply:
Pero, si lo usan como una clase al igual que los modelos, deben cambiarlo a mayuscula, si es una variable simple ps ok, pero lo dudo xd